### PR TITLE
Add initial governance section

### PR DESCRIPTION
**Pull request by XAMPPRocky**
r? @nikomatsakis 

cc @rust-lang/wg-governance

### CODE REVIEW

File: src/governance/README.md [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+
+- **Sets the overall direction and vision for the project.** That means setting
+  the core values that are used when making decisions about technical
+  tradeoffs. It means steering the project toward specific use cases where Rust
+  can have a major impact. It means leading the discussion, and writing RFCs
+  for, _major_ initiatives in the project.
+
+- **Sets the priorities and release schedule.** Design bandwidth is limited, and
+  it's dangerous to try to grow the language too quickly; the core team makes
+  some difficult decisions about which areas to prioritize for new design, based
```
Comment (Centril) [inline src/governance/README.md]: The compiler, libs, and lang teams decide what design and implementation areas to focus on autonomously.

File: src/governance/README.md [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+## Core team
+
+**The core team serves as leadership for the Rust project as a whole**. In
+particular, it:
+
+- **Sets the overall direction and vision for the project.** That means setting
+  the core values that are used when making decisions about technical
+  tradeoffs. It means steering the project toward specific use cases where Rust
+  can have a major impact. It means leading the discussion, and writing RFCs
+  for, _major_ initiatives in the project.
```
Comment (Centril) [inline src/governance/README.md]: This doesn't seem true anymore if it refers to major language design RFCs.

File: src/governance/README.md [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+
+- **Focuses on broad, cross-cutting concerns.** The core team is specifically
+  designed to take a _global_ view of the project, to make sure the pieces are
+  fitting together in a coherent way.
+
+- **Spins up or shuts down subteams.** Over time, we may want to expand the set
+  of subteams, and it may make sense to have temporary "strike teams" that focus
+  on a particular, limited task.
+
+- **Decides whether/when to ungate a feature.** While the subteams make
```
Comment (Centril) [inline src/governance/README.md]: This is definitely not true anymore.

File: src/governance/README.md [hunk 4]
```diff
@@ -0,0 +1,428 @@
...
+in addition have demonstrated exceptional communication, design, and people
+skills. They must be able to work with a diverse group of people and help lead
+it toward consensus and execution.
+
+Each subteam is led by a member of the core team. **The leader is responsible for**:
+
+- Setting up the subteam:
+
+  - Deciding on the initial membership of the subteam (in consultation with
+    the core team). Once the subteam is up and running.
```
Comment (jonas-schievink) [inline src/governance/README.md]: That last sentence seems out of place

File: src/governance/README.md [hunk 5]
```diff
@@ -0,0 +1,428 @@
...
+  consensus otherwise (should be rare).
+
+The way that subteams communicate internally and externally is left to each
+subteam to decide, but:
+
+- Technical discussion should take place as much as possible on public forums,
+  ideally on RFC/PR threads and tagged discuss posts.
+
+- Each subteam will have a dedicated
+  [discuss forum](http://internals.rust-lang.org/) tag.
```
Comment (jonas-schievink) [inline src/governance/README.md]:
```suggestion
  [internals forum](https://internals.rust-lang.org/) tag.
```

File: src/governance/README.md [hunk 6]
```diff
@@ -0,0 +1,428 @@
...
+  shepherds/reviewers as needed.
+
+- Making final decisions in cases of contentious RFCs that are unable to reach
+  consensus otherwise (should be rare).
+
+The way that subteams communicate internally and externally is left to each
+subteam to decide, but:
+
+- Technical discussion should take place as much as possible on public forums,
+  ideally on RFC/PR threads and tagged discuss posts.
```
Comment (jonas-schievink) [inline src/governance/README.md]: "tagged discuss posts"?

File: src/governance/README.md [hunk 7]
```diff
@@ -0,0 +1,428 @@
...
+
+- Subteams should have some kind of regular meeting or other way of making
+  decisions. The content of this meeting should be summarized with the rationale
+  for each decision -- and, as explained below, decisions should generally be
+  about weighting a set of already-known tradeoffs, not discussing or
+  discovering new rationale.
+
+- Subteams should regularly publish the status of RFCs, PRs, and other news
+  related to their area. Ideally, this would be done in part via a dashboard
+  like [the Homu queue](http://buildbot.rust-lang.org/homu/queue/rust)
```
Comment (jonas-schievink) [inline src/governance/README.md]:
```suggestion
  like [the Homu queue](http://buildbot.rust-lang.org/homu/queue/rust).
```

File: src/governance/README.md [hunk 8]
```diff
@@ -0,0 +1,428 @@
...
+feel a strong need to _actively block_ the RFC from progressing.
+
+One important question is: consensus among which people, exactly? Of course, the
+broader the consensus, the better. But at the very least, **consensus within the
+members of the subteam should be the norm for most decisions.** If the core team
+has done its job of communicating the values and priorities, it should be
+possible to fit the debate about the RFC into that framework and reach a fairly
+clear outcome.
+
+[consensus]: http://en.wikipedia.org/wiki/Consensus_decision-making
```
Comment (jonas-schievink) [inline src/governance/README.md]:
```suggestion
[consensus]: https://en.wikipedia.org/wiki/Consensus_decision-making
```

**Comment by XAMPPRocky** [comment]
@Centril We're very much aware of the staleness of this document. This is currently solely the "Detailed Design" section of RFC 1068, with no changes made. I should have given more context but right now this PR is more "Is this the right place to put this in forge?", and we'll make actual content changes in separate PRs later so it's easier to review.

**Pull request merged**